### PR TITLE
[DOPS-895] Feature/ci update

### PR DESCRIPTION
**Pull request by Cre-eD**
# Task
[DOPS-895] Create new CI for new D3 repos.
## Changes
1. CI updated.

## Author
Signed-off-by: Dmitriy Creed <creed@soramitsu.co.jp>

[DOPS-895]: https://soramitsu.atlassian.net/browse/DOPS-895

### CODE REVIEW

**Review by Mingela — APPROVED** [review]

**Review by ravenolf — APPROVED** [review]

**Review by kogeler — APPROVED** [review]
OK

File: Jenkinsfile [hunk 1]
```diff
@@ -7,30 +7,13 @@ pipeline {
     skipDefaultCheckout()
```
Comment (BulatSaif) [inline Jenkinsfile]:
```suggestion
```

File: Jenkinsfile [hunk 2]
```diff
@@ -7,30 +7,13 @@ pipeline {
...
-            builds.cancelSameJobBuilds()
-          }
-        }
-      }
-    }
     stage('Tests') {
-      agent { label 'd3-build-agent' }
       steps {
         script {
           def scmVars = checkout scm
```
Comment (BulatSaif) [inline Jenkinsfile]:
```suggestion
```
replace `scmVars.` with `env.`

File: Jenkinsfile [hunk 3]
```diff
@@ -108,18 +91,16 @@ pipeline {
     }
 
     stage('Build and push docker images') {
-      agent { label 'd3-build-agent' }
       steps {
         script {
           def scmVars = checkout scm
```
Comment (BulatSaif) [inline Jenkinsfile]:
```suggestion
```
replace `scmVars.` with `env.`

File: deploy/docker-compose-single.yml [hunk 4]
```diff
@@ -26,7 +26,7 @@ services:
       - d3-network
 
   d3-changelog:
-    image: nexus.iroha.tech:19002/d3-deploy/changelog-endpoint:master
+    image: nexus.iroha.tech:19004/soranet/changelog-endpoint:master
```
Comment (Mingela) [inline deploy/docker-compose-single.yml:29]: its not yet there

**Review by Mingela — APPROVED** [review]

**Review by ravenolf — APPROVED** [review]

File: deploy/docker-compose-single.yml (hunk 4 above)
Comment (kogeler) [reply deploy/docker-compose-single.yml:29]: If you want to use two registry in tests you have to add the step to login for both.

**Review by kogeler — APPROVED** [review]
Ok. But see the comment

**Review by BulatSaif — APPROVED** [review]

**Pull request merged**